### ISSUE TITLE
List successful tests

### ISSUE THREAD

**Issue by phidias** [krajzeg/chaplain#6]
Hi,

I started using chaplain and I like it very much. It was very easy to get started. What I found that I'm missing now is a list of the successful tests. Here is an example below (this is sbt scala tests).

![chaplain-success-example](https://cloud.githubusercontent.com/assets/301011/15938870/54e9f8ce-2e75-11e6-93a5-3ba01b75b304.jpg)

Basically, the successful tests allow you to verify that you covered all the cases (urls in this case). Also it gives you the satisfaction looking at the green colors covering your console .. hehe.

Is there any chance you can do something like that? It can be optional with a `-v` if you don't like it so much.

**Comment by krajzeg** [comment]
This actually _was_ how chaplain worked at first, but I quickly found it cumbersome with a larger number of tests . But I've also used mocha with the spec reporter and 200+ tests in the past, which looks mostly like your screenshot and scrolls past too quickly to read, so it's mostly a question of taste and what you're used to :).

Let's just say this is not a high priority - most of the work until 0.4.0 will be focused on getting the test themselves as generic and easy as possible. I hope this is not a deal breaker for you. 

I'll keep this issue open and deal with it some time before 0.5.0, most probably by a "reporter" system like [Mocha's](https://mochajs.org/#reporters).

**Comment by phidias** [comment]
Yeah sure. It's not a big thing this one. I can do my verification by going though the configuration file. But will be good to have one day.

Indeed, when you run all the tests it's too much to output, but when you run a smaller part of it (was just going to create a new issue about it) .. let's say a controller, then the output will be smaller and very helpful (to see if you covered everything).

**Comment by phidias** [comment]
Not creating a new issue .. found how you can run all the tests of a certain suite (`chaplain run drinks.*`)
